Microsoft.Toolkit.Uwp.UI.Controls.Input/Microsoft.Toolkit.Uwp.UI.Controls.Input.csproj

```diff
@@ -13,7 +13,18 @@
         - RemoteDevicePicker: Remote Device Picker Control for Project Rome.
         - TokenizingTextBox: An AutoSuggestBox like control which places entered input into easily removed containers for contacts or tags.
     </Description>
-    <PackageTags>UWP Toolkit Windows Controls XAML Range Radial Gauge RadiaGauge Tokenizing TextBox</PackageTags>
+    <PackageTags>
+        Windows Community Toolkit;Windows;Community;Toolkit;WCT;UWP;Controls;XAML;
+        Color Picker Button ;ColorPickerButton ;Button    ;
+        ColorPicker  Button ;ColorPicker       ;
+        Radial Gauge        ;RadialGauge       ;Radial    ;Guage   ;
```

Comment:
Is the extra whitespace going to be ignored?

Reply:
Going to test it now.

Reply:
It did not seem to cause any problems 
e.x.
`Controls.Input` results in this in VS package explore.
`Windows, Community, Toolkit, Windows, Community, Toolkit, WCT, UWP, Controls, XAML, Color, Picker, Button, ColorPickerButton, Button, ColorPicker, Button, ColorPicker, Radial, Gauge, RadialGauge, Radial, Guage, Range, Selector, RangeSelector, Range, Selector, Remote, Device, Picker, RemoteDevicePicker, Remote, Remote, DevicePicker, DevicePicker, RemoteDevice, Picker, RemoteDevice, Tokenizing, Text, Box, TokenizingTextBox, Tokenizing, Tokenizing, TextBox, TextBox`

It seems like multi word tags dont work in nuget so it seems like having the multiple lines and the spacing does not cause any additional issues.